merlin-sdk/src/main/java/gov/nasa/jpl/aerie/merlin/protocol/types/DurationType.java

```diff
@@ -1,5 +1,6 @@
 package gov.nasa.jpl.aerie.merlin.protocol.types;
 
 public sealed interface DurationType {
+  record Controllable(String parameterName) implements DurationType {}
   record Uncontrollable() implements DurationType {}
 }
```

Comment:
I need to get comfortable with this pattern. Sealed interfaces and records still feel pretty new to me, so I tend to not think of them in my own designs too much, but this seems to work well.